### PR TITLE
Added before script

### PR DESCRIPTION
**Pull request by lightningspirit**
This PR adds a env var to configure a before run script.
It's useful in the situation you want to measure performance or send events to an event stream like Kafka.

P.S. thank you for this very useful Docker image @woolfg 🥇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The following environment variables are supported (incl. example values):
 - `DIR_DATE_PATTERN:` "%Y%m%d"
 - `FULL_BACKUP_DATE_FORMAT:` "%a"
 - `FULL_BACKUP_DATE_RESULT:` "Sun"
+- `BEFORE_BACKUP_SCRIPT:` "/backup/before_script.sh"
```
Comment (woolfg) [inline README.md:25]: could you please add the information about the first parameter that you hand over to the before script.

**Review by woolfg — CHANGES_REQUESTED** [review]
thanks a lot for the PR, looks good. could you just add the requested information to the readme, then I'll merge it. thanks for the contribution :green_heart:

File: README.md (hunk 1 above)
Comment (lightningspirit) [reply README.md:25]: Sure thing, no problem, I can do it 😉

**Comment by lightningspirit** [comment]
Hi @woolfg 👋 I just added a section in README about the after and before scripts. Is this ok for you?

**Comment by woolfg** [comment]
thanks for the contribution, looks good

**Pull request merged**